src/main/webapp/app/tasks/task.ts

```diff
@@ -4,4 +4,5 @@
 export interface Task {
   id: string;
   name: string;
+  done?: boolean;
```

Comment:
I'd not go for an optional property here as it introduces some weird ternary logic on the done state